xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-test/xwiki-platform-image-lightbox-test-docker/src/test/it/org/xwiki/image/lightbox/test/ui/LightboxIT.java

```diff
@@ -46,24 +46,24 @@
 
 /**
  * Functional tests for the image lightbox.
- * 
+ *
  * @version $Id$
  * @since 14.1RC1
  */
 @UITest(properties = {
     // Add the FileUploadPlugin which is needed by the test to upload attachment files
-    "xwikiCfgPlugins=com.xpn.xwiki.plugin.fileupload.FileUploadPlugin"})
+    "xwikiCfgPlugins=com.xpn.xwiki.plugin.fileupload.FileUploadPlugin" })
 class LightboxIT
 {
+    public static final String USER_NAME = "JohnDoe";
```

Comment:
why public BTW?

Reply:
Created it by mistake as public, it was just moved by `reformat` here
I modified it to use private now